### PR TITLE
feat(multi-select): add selected preview

### PR DESCRIPTION
**Pull request by Daryl-L**
https://github.com/Magickbase/ckb-explorer-public-issues/issues/721

image

image

### CODE REVIEW

**Comment by FrederLu** [comment]
image
image

Please adjust the font style of `Symbol&Name` according to the UI style.

File: src/pages/NftCollections/List.tsx [hunk 1]
```diff
@@ -49,43 +49,43 @@ function useFilterList(): Record<'title' | 'value', string>[] {
 const filterList = [
   {
     key: 'invalid',
-    value: 'invalid',
+    value: 'Invalid',
```
Comment (Keith-CY) [inline src/pages/NftCollections/List.tsx:52]: It's better to use `i18n`

File: src/pages/Xudts/index.tsx [hunk 2]
```diff
@@ -39,19 +39,19 @@ const filterList = [
   // },
   {
     key: 'layer-1-asset',
-    value: 'layer-1-asset',
+    value: 'Layer 1 Asset',
```
Comment (Keith-CY) [inline src/pages/Xudts/index.tsx:42]: It's better to use `i18n`

Comment (Keith-CY) [reply src/pages/Xudts/index.tsx:42]: Any update on this conversation?

**Comment by FrederLu** [comment]
https://github.com/user-attachments/assets/12bd4191-969d-433d-87e7-222a2d95d7f3

1. After selecting Chinese on the page, it will automatically return to English when selecting Tags.
2. Added the function of showing that the selected tag of Tags lacks Chinese (when the tags displayed on the page are in Chinese).

**Review by FrederLu — APPROVED** [review]

**Review by Keith-CY — APPROVED** [review]

**Pull request merged**